requirements.txt

```diff
@@ -9,7 +9,6 @@ python-rake>=1.0.7
 sklearn
 scipy==0.18.0
 keras==1.2.1
-tensorflow==0.12.1
+tensorflow==1.1.0
```

Comment:
`0.12.1` is no longer on pypi. works fine on my local with `1.1.0` (at least the parts i'm running) - i'd already say that's lesser of two breaks.